packages/components/Shimmer/FluentUIReactNativeShimmer.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.author           = { "Microsoft" => "fluentuinativeowners@microsoft.com"}
   s.source           = { :git => "https://github.com/microsoft/fluentui-react-native.git", :tag => "#{s.version}" }
   s.swift_version    = "5"
-  s.module_name      = 'FluentUIReactNative'
+  s.module_name      = 'FluentUIReactNativeShimmer'
```

Comment:
Probably makes sense for this to be in a separate PR as this is for the Shimmer control, not the Spinner control

Reply:
Good point. I didn't realize two things can't have the same module name hence this change. I can make a separate Shimmer PR to change some things and merge that in first